### ISSUE TITLE
run-help --app error text

### ISSUE THREAD

**Issue by 15baraniana** [apptainer/singularity#3174]
## Version of Singularity:

3.1-2

### Actual behavior

```
singularity run-help --app foo rand.sif 
WARNING: Authentication token file not found : Only pulls of public images will succeed
No help sections were defined for this image
```

When there is no `%help` section in the recipe file, this error is appropriate but it also occurs when there isn't the specified `%apphelp` section. I think it would be clearer to change the error message to "No help section defined for foo for this image"

Also, is the warning necessary? It already warns you when you build the image, I don't think it needs to warn you in the run-help command.

**Comment by jscook2345** [comment]
@ikaneshiro 

What is the status on the warning being removed from commands? Will that be a part of the `remote` work?

**Comment by ikaneshiro** [comment]
@jscook2345 Plan is for that to be gone in 3.2 with the `remote` work, that wont be included in 3.1.x though because we're changing some of the CLI

**Comment by WestleyK** [comment]
This issue is not relevant anymore, the `remote` command PR has been merged (https://github.com/sylabs/singularity/pull/2829).
